includes/discovery/os/swos.inc.php

```diff
@@ -1,7 +1,10 @@
 <?php
 
 if (!$os) {
-    if (is_numeric(snmp_get($device, 'SNMPv2-SMI::enterprises.14988.2', '-Oqv', ''))) {
-        $os = 'swos';
+    if (strstr(snmp_get($device, 'SNMPv2-MIB::sysName.0', '-Oqv', ''), 'MikroTik')) {
+        $os = 'routeros';
```

Comment:
Slight change of plan, remove `$os = 'routeros';` as we will still then run through the routeros discovery but it will still then go on to pick up swos in this file.